<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PigsBoxInfoDialog</class>
 <widget class="QDialog" name="PigsBoxInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>656</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>443</width>
    <height>656</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>443</width>
    <height>656</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QWidget {font-family: arial;font-size: 32px;}</string>
     </property>
     <property name="text">
      <string>Box #1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resouces.qrc">:/assets/cute_pig_bun35.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">QLabel {font-family: arial; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Box hall:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true">QLabel {font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Box type:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">QLabel {font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Pigs number:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">QLabel {font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Population date:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">QLabel {font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Release date:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">QLabel {font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Dimensions:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resouces.qrc"/>
 </resources>
 <connections/>
</ui>
